docs/getting-started-guides/network-policy.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caseydavenport
+
+---
+
+Kubernetes can be used to declare network policies which govern how Pods can communicate with each other.  This document helps you get started using the Kubernetes [NetworkPolicy API](/docs/user-guide/networkpolicies), and provides a demonstration thereof. 
+
+In this article we assume that that a Kubernetes cluster has been created with network policy support.  It's easy to create such a cluster using the [GCE guide](/docs/getting-started-guides/gce), setting the `NETWORK_POLICY_PROVIDER=calico` environment variable.
+
+e.g
+
+```shell
+export NETWORK_POLICY_PROVIDER=calico
+curl -sS https://get.k8s.io | bash
+```
+
+## Using NetworkPolicy 
+
+Once your cluster is started, you should see a collection of pods running in the `kube-system` Namespace that support Kubernetes NetworkPolicy.
+
+```shell
```

Comment:
I think you can use `console` highlighter here instead, I've seen it used in other docs.